dtmi/innodisk/innoage/devcmd-1.json

```diff
@@ -0,0 +1,102 @@
+{
+    "@context": [
+        "dtmi:iotcentral:context;2",
```

Comment:
consider removing the `iotcentral:context` if you are not using it.

Reply:
No, in our business, we also support iotcentral, if customer need.